torch/_inductor/scheduler.py

```diff
@@ -213,7 +213,7 @@ def allocate(self):
                 # o what have i done.  lets make this an api
                 or (
                     isinstance(self, ExternKernelSchedulerNode)
-                    and isinstance(self.node, ir.AllReduce)
+                    and isinstance(self.node, (ir.AllReduce, ir.ForceInPlace))
```

Comment:
hope to remove the allreduce case after unifying other collectives w/ using ForceInPlace (and renaming it) - ok in future PR